HTML documentation isn't generated

I am working to update the Fedora package to 2.8.0. Unfortunately, it looks like the HTML documentation is no longer generated properly. Running `make html` in the build directory `objdir/`, I observe that the build system compiles `progman.pdf` and then

```
/usr/bin/latex2html progman
This is LaTeX2HTML Version 2023.2 (Released July 1, 2023)
...producing markup for HTML version 5.0  



Extension: loading /usr/share/latex2html/versions/unicode.pl
*** Constructing conversion utf-8 -> Unicode ***

HTML version: loading /usr/share/latex2html/versions/html5_0.pl

 *** processing declarations ***
Cannot read ./progman 
make[2]: *** [/home/susi/fedora/libint2/rawhide/libint-2.8.0/doc/progman/Makefile:81: html] Error 2
make[2]: Leaving directory '/home/susi/fedora/libint2/rawhide/libint-2.8.0/objdir/doc/progman'
make[1]: *** [/home/susi/fedora/libint2/rawhide/libint-2.8.0/doc/Makefile:58: html] Error 1
make[1]: Leaving directory '/home/susi/fedora/libint2/rawhide/libint-2.8.0/objdir/doc'
make: *** [/home/susi/fedora/libint2/rawhide/libint-2.8.0/Makefile:128: html] Error 1
```

--- a/configure.ac
+++ b/configure.ac
@@ -1117,8 +1117,30 @@ AC_PATH_PROG(PDFLATEX,pdflatex,,$PATH:/usr/local/bin:/opt/local/bin)
 AC_PATH_PROG(PSLATEX,pslatex,,$PATH:/usr/local/bin:/opt/local/bin)
 AC_PATH_PROG(DVIPS,dvips,,$PATH:/usr/local/bin:/opt/local/bin)
 AC_PATH_PROG(LATEX,latex,,$PATH:/usr/local/bin:/opt/local/bin)
-AC_PATH_PROG(LATEX2HTML,latex2html,,$PATH:/usr/local/bin:/opt/local/bin)
 AC_PATH_PROG(BIBTEX,bibtex,,$PATH:/usr/local/bin:/opt/local/bin)
+
+CHECK_FOR_LATEX2HTML=no
+AC_ARG_ENABLE(latex2html,
+AS_HELP_STRING([--enable-latex2html],[Enable detection of latex2html (broken on many platforms, hence disabled by default)]),
+[
+case $enableval in
+  yes)
+    CHECK_FOR_LATEX2HTML=yes
+  ;;
+  no)
+    CHECK_FOR_LATEX2HTML=no
+  ;;
+  *)
+    AC_MSG_ERROR([Invalid value for --enable-latex2html ($enableval)])
+  ;;
+esac
+],[
+    CHECK_FOR_LATEX2HTML=no
+]
+)
+if [ X$CHECK_FOR_LATEX2HTML = Xyes ]; then
+  AC_PATH_PROG(LATEX2HTML,latex2html,,$PATH:/usr/local/bin:/opt/local/bin)
+fi
 
 dnl The lack of certain tools is a show stopper
 

